ui/src/types/jupyterhub.ts

```diff
@@ -18,6 +18,11 @@ export interface JhApp {
   username?: string;
   ready: boolean;
   public: boolean;
+  status: {
+    stopped: boolean;
+    pending: any; // Adjust according to your actual type
+    ready: boolean;
+  };
```

Comment:
Probably doesn't need to be an object. Might be easier to handle just as a string value, based on all of these